doc/start/hardware-recommendations.rst

```diff
@@ -447,10 +447,11 @@ Additional Considerations
 Ceph operators typically provision  multiple OSDs per host, but you should
 ensure that the aggregate throughput of your OSD drives doesn't exceed the
 network bandwidth required to service a client's read and write operations.
-You should also each host's percentage of the cluster's overall capacity. If
-the percentage located on a particular host is large and the host fails, it
-can lead to problems such as recovery causing OSDs to exceed the ``full ratio``,
-which in turn causes Ceph to halt operations to prevent data loss.
+You should also consider each host's percentage of the cluster's overall
```

Comment:
Good catch.

Reply:
My shame is without bound (I wrote these sentences).